File: jasmine-standalone-2.4.1/src/Plane.js
function Plane() {
  this.status = false;
};

Plane.prototype.showstatus = function(){
  return this.status;
}

Plane.prototype.land = function(){
  this.status = true;
  return this.status;
}

Plane.prototype.takeoff = function(){
  this.status = false;
  return this.status;
}

File: plane.rb
class Plane
  attr_reader :landed
  def initialize(landed = false)
    @landed = landed
  end

  def land
    unless @landed
      @landed = true
      return true
    end
    false
  end

  def takeoff
    if @landed
      @landed = false
      return true
    end
    false
  end
end